scripts/apps/contacts/components/Form/ContactNumberInput.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Row, LineInput, Input, SelectInput, Toggle, Label} from './';
```

Comment:
would consider moving this to `core/components/form` so we can reuse this later on